torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -2992,6 +3040,36 @@ def _test_all_to_all_single_unequal_split_helper(
                 self.assertEqual(out_tensor, expected_tensor)
             self._barrier()
 
+        def _test_all_to_all_helper_quantizer(
```

Comment:
Isn't it simpler to just extend `_test_all_to_all_helper` instead of having another method?